features/nanostack/mbed-mesh-api/source/ethernet_tasklet.c

```diff
@@ -283,7 +300,7 @@ void enet_tasklet_init(void)
     }
 }
 
-int8_t enet_tasklet_network_init(int8_t device_id)
+uint8_t enet_tasklet_network_init(int8_t device_id)
```

Comment:
Correction should be the other way, I believe - it's the header that's wrong. `network_interface_id` is `int8_t`.

#8698 is already correcting this that way - this would conflict. Either don't touch, or make an identical change to it.